io.github.Abstract_AA.Framestop.yml

```diff
@@ -0,0 +1,33 @@
+id: io.github.Abstract_AA.Framestop
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+command: framestop
+finish-args:
+  - --share=network
+  - --filesystem=home
+  - --socket=wayland  # For Wayland display server
+  - --socket=fallback-x11  # For X11 display server
+  - --share=ipc
+  # - --talk-name=org.freedesktop.DBus  # D-Bus permissions
+  - --device=dri  # For hardware acceleration
+
+modules:
+  - name: Framestop
+    buildsystem: simple
+    build-commands:
+      # Install the Python script to the correct location, update this
+      - install -D framestop.py /app/bin/framestop
+      - install -D io.github.Abstract_AA.Framestop.svg /app/share/icons/hicolor/scalable/apps/io.github.Abstract_AA.Framestop  # Icon must match app ID
+      - install -D io.github.Abstract_AA.Framestop.desktop /app/share/applications/io.github.Abstract_AA.Framestop.desktop  # .desktop file
+    sources:
+      - type: dir
+        path: src/
+
+  - name: python3-dependencies
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-index --find-links=src/packages/ --ignore-installed moviepy-1.0.3.tar.gz pillow-10.4.0-cp311-cp311-manylinux_2_28_x86_64.whl imageio-2.35.1-py3-none-any.whl imageio_ffmpeg-0.5.1-py3-none-manylinux2010_x86_64.whl decorator-4.4.2-py2.py3-none-any.whl tqdm-4.66.5-py3-none-any.whl requests-2.32.3-py3-none-any.whl numpy-2.1.1-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl charset_normalizer-3.3.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl idna-3.8-py3-none-any.whl urllib3-2.2.2-py3-none-any.whl certifi-2024.8.30-py3-none-any.whl proglog-0.1.10-py3-none-any.whl setuptools-74.1.2-py3-none-any.whl basic_colormath-0.4.0-py3-none-any.whl
+    sources:
+      - type: dir
+        path: src/packages/
```

Comment:
this is not how you specify python dependencies.

See https://github.com/flatpak/flatpak-builder-tools/tree/master/pip